subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/AggregatingIncrementalAnnotationProcessingIntegrationTest.groovy

```diff
@@ -254,7 +253,7 @@ class AggregatingIncrementalAnnotationProcessingIntegrationTest extends Abstract
         run "compileJava"
 
         then:
-        outputs.recompiledFiles("A", "ServiceRegistry", "ServiceRegistryResource.txt")
+        outputs.recompiledFiles("A", "ServiceRegistry", "ServiceRegistryResource.txt", "Dependent")
```

Comment:
This assert had been broken a while ago and was missed during review. The test was no longer asserting what its descriptions says. I only noticed this because my simplifications fixed the behavior again.